database_wrapper.py

```diff
@@ -10,9 +10,11 @@
     ents = db.get_entities()
 """
 import json
+import csv
```

Comment:
smart! I would've been lazy with `pandas`, but `import csv` makes more sense and is simpler!